src/local/butler/lint.py

```diff
@@ -185,7 +191,7 @@ def execute(_):
   py_changed_file_paths = [
       f for f in file_paths if f.endswith('.py') and
       # Exclude auto-generated files.
-      not f.endswith('_pb2.py') and not f.endswith('_pb2_grpc.py')
+      not auto_generated(f)
```

Comment:
nit: can remove the comment on line 193 as the function name is self-descriptive `is_auto_generated`

Reply:
Done